### ISSUE TITLE
Allow Channel Chaining to allow better Signal Flow

### ISSUE THREAD

**Issue by Anonymouqs** [LMMS/lmms#4195]
Can we have support for an effects channel to send to another channel instead of a master channel?
This would allow us to:
* Apply effects in a more efficient manner. By allowing us to apply effects to groups of groups of signals.
* Organize signals better.
* For example, someone might want all of the bass to have a certain effect applied to them, but then individual groups of the bass might have certain effects applied to them.

For example, have channels 1,2,3 sent to channel 4, which would be sent to the Master Channel(by default)

I believe the implementation would be easy as the code is overall within the program:
* Use the code from the built-in VSTs to choose a certain channel to send the signal to.
* Applying the master channel code in able to allow a channel to receive another channel.
* The only thing we really need to code is a circular dependency detector if it crashes the program. If it doesn't, in theory, those channels would be silent. But I do believe that just dependencies would crash LMMS. 
* By default, channels would send to master.

**Comment by Sawuare** [comment]
This is already implemented in LMMS 1.2.0-rc.5.
![image](https://user-images.githubusercontent.com/25503477/36680648-6c6d450a-1b27-11e8-8b2a-2acc77a2ced8.png)

**Comment by tresf** [comment]
Yes, everything including circular detection are in 1.2.0.  Closing.

https://github.com/LMMS/lmms/commit/d9d085d14e8258340d61f3e2402ee05bc675cbd4

**Comment by Anonymouqs** [comment]
How do I use the send feature? The documentation doesn't help me at all.

**Comment by Anonymouqs** [comment]
Apologies, I found directions in the documentation. Can we add tooltips to the send channels? It's a bit confusing at first glance and would enhance user-friendliness.

**Comment by zonkmachine** [comment]
> Can we add tooltips to the send channels?

There is a PR for this against master. https://github.com/LMMS/lmms/pull/4128
There is currently info (what's this text) available but no tooltip. Right click the `send` label and select 'Help' (bottom).

**Comment by tresf** [comment]
@Anonymouqs, in addition to the PR that @zonkmachine linked, @raindropsfromsky has proposed a more intuitive UI here: https://github.com/LMMS/lmms/issues/4187, https://github.com/LMMS/lmms/issues/4181.
